Gemfile

```diff
@@ -94,7 +94,7 @@ gem 'pry-rails', require: false
 gem 'r2', '~> 0.2.5', require: false
 gem 'rake'
 
-
+gem 'faraday_middleware'
```

Comment:
Is there a reason why you need `faraday_middleware` here? 

We already have [Excon](https://github.com/excon/excon) :)